tests/test_intervention_outcome.py

```diff
@@ -0,0 +1,160 @@
+from conftest import *
+from datetime import datetime
+
+from models.appendix import InterventionReason
+from models.prescription import Intervention
+from security.role import Role
+
+import json
+from collections.abc import Mapping
+
+# import auxiliary dictionaries to this test
+from tests.aux_test_intervention import *
+
+
+def get_nested_value(data, path):

```

Comment:
se a função não for um teste e for usada somente dentro deste módulo, iniciar com _
Exemplo: _get_nested_value
Este é um padrão que indica que a função é interna

**aplicar para todos os casos onde isso acontece

Reply:
changed